Wikipedia/Code/EditSummaryViewController.swift

```diff
@@ -114,3 +109,10 @@ extension EditSummaryViewController: UITextFieldDelegate {
         return newLength <= EditSummaryViewController.maximumSummaryLength
     }
 }
+
+public class SummaryButtonScrollView: UIScrollView {
+    override public var intrinsicContentSize: CGSize {
+        subviews.first?.layoutIfNeeded()
```

Comment:
`subviews.first` could be switched out from under you to be a different view then the one you're looking for, so this should be an explicit reference. Also, `intrinsicContentSize` shouldn't have a layout pass in it or [depend on the size of the view](https://developer.apple.com/documentation/uikit/uiview/1622600-intrinsiccontentsize) (it's done elsewhere in the codebase, but it's also problematic there). Here, you could add an outlet collection for the buttons and get the max intrinsic content height to use for the scroll view's intrinsic height. The width can be `UIView.noIntrinsicMetric`. `AutoLayoutSafeMultiLineButton` could be updated to use `UIView.noIntrinsicMetric` instead of `bounds.size.width` in the future, but not worth the impact for 6.2

Reply:
fixing...

Reply:
fixed!